tests/utils.py

```diff
@@ -72,7 +72,7 @@ def make_venv(venv_dir: Path) -> VenvInfo:
         if "ensurepip" in e.cmd and b"KeyboardInterrupt" not in e.stdout.splitlines():
             print_error(
                 "stubtest requires a Python installation with ensurepip. "
-                "If on Linux, you may need to install the python3-venv package."
+                + "If on Linux, you may need to install the python3-venv package."
```

Comment:
Maybe we should just turn off the pyright rule against implicit string concatenation?